modules/auxiliary/scanner/ssh/cerberus_sftp_enumusers.rb

```diff
@@ -81,7 +81,10 @@ def check_vulnerable(ip)
     end
 
     auth = Net::SSH::Authentication::Session.new(transport, opt_hash)
-    auth.authenticate("ssh-connection", Rex::Text.rand_text_alphanumeric(8), Rex::Text.rand_text_alphanumeric(8))
+    begin
+      auth.authenticate("ssh-connection", Rex::Text.rand_text_alphanumeric(8), Rex::Text.rand_text_alphanumeric(8))
+    rescue NoMethodError
```

Comment:
Empty-rescuing errors like this is dangerous, since it's hard to debug. Until this is fixed upstream, consider putting a `print_error` in.

Reply:
Good call. I'll take on this PR and I will make sure to follow this advice. Thanks!